com.polyphonesoundfonts.polyphone.json

```diff
@@ -0,0 +1,156 @@
+        "--enable-shared"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "http://www.music.mcgill.ca/~gary/rtmidi/release/rtmidi-4.0.0.tar.gz",
+          "sha256": "370cfe710f43fbeba8d2b8c8bc310f314338c519c2cf2865e2d2737b251526cd"
+        }
+      ]
+    },
+    {
+      "name": "stk",
+      "build-commands": [
+        "make -j $FLATPAK_BUILDER_N_JOBS -C src",
+        "make -j $FLATPAK_BUILDER_N_JOBS -C projects/effects libeffects",
+        "make -j $FLATPAK_BUILDER_N_JOBS -C projects/ragamatic libragamat",
+        "make -j $FLATPAK_BUILDER_N_JOBS -C projects/eguitar libeguitar"
+      ],
+      "post-install": [
+        "sh ./stk-install.sh"
+      ],
+      "cleanup": [
+        "/bin/*",
+        "*.a"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "http://ccrma.stanford.edu/software/stk/release/stk-4.6.1.tar.gz",
+          "sha256": "e77ba3c80cdd93ca02c34098b9b7f918df3d648c87f1ed5d94fe854debd6d101"
+        },
+        {
+          "type": "file",
+          "path": "stk-install.sh"
+        }
+      ]
+    },
+    {
+      "name": "qcustomplot",
+      "buildsystem": "qmake",
+      "config-opts": [
+        "LIBDIR=/app/lib"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://www.qcustomplot.com/release/2.0.1/QCustomPlot-source.tar.gz",
+          "sha256": "574cee47def3251d080168a23428859214277cb9b6876bcdb9ce9d00b2403fe4"
+        },
+        {
+          "type": "file",
+          "path": "patches/qcustomplot.pro"
+        }
+      ]
+    },
+    {
+      "name": "polyphone",
+      "buildsystem": "qmake",
+      "post-install": [
+        "make install",
+        "install bin/polyphone /app/bin",
+        "install contrib/polyphone.svg -D -t /app/share/icons/hicolor/scalable/apps",
+        "install resources/polyphone.png -D -t /app/share/icons/hicolor/512x512/apps",
+        "install contrib/polyphone.desktop -D -t /app/share/applications",
+        "install contrib/polyphone.xml -D -t /app/share/mime/packages",
+        "install contrib/*.1 -D -t /app/share/man/man1",
+        "install README changelog -D -t /share/doc/polyphone",
+        "install com.polyphonesoundfonts.polyphone.appdata.xml -D -t /app/share/metainfo"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "http://www.polyphone-soundfonts.com/download/file/689-polyphone-2-2-zip/latest/download?fd7d8db8ad3ed1d98cf0b5a73ffe2a46=1&return=aHR0cHMlM0ElMkYlMkZ3d3cucG9seXBob25lLXNvdW5kZm9udHMuY29tJTJGZG93bmxvYWQ=",
+          "sha256": "ac6c9f39c10356e44e993f1d3c4454dd8dbdc5e0b5ee3ca387009b4edb0dd269",
+          "dest-filename": "polyphone-2-2.zip"
+        },
+        {
+          "type": "patch",
+          "path": "polyphone-make.patch",
+          "strip-components": 2
+        },
+        {
+          "type": "file",
+          "path": "com.polyphonesoundfonts.polyphone.appdata.xml"
```

Comment:
New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html

Reply:
done.